doc/DeveloperGuide.md

```diff
@@ -94,6 +94,12 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be carry out operations within O(log N) time
```

Comment:
Recall from tutorial that NFRs should be in layman terms, the people reading the NFRs may not know what is `O(lg N)` time =).

Reply:
Yes, so something  along the lines of:
5. Should respond to user input within 1 second.